main.py

```diff
@@ -63,17 +65,37 @@
         m = market.Market(0, network=sc.power_network)
 
     list_ts = linspace(cfg.start, cfg.start + cfg.nb_ts - 1, cfg.nb_ts)
+
+    # Actors generate their order prices from the market maker prices. These need to be the same
+    # for each actor. In the future market maker prices should be stored in a single data structure,
+    # eg. market object, a market maker instance or something similar.
+    for a in sc.actors:
+        a.data.selling_price = sc.actors[0].data.selling_price
+        a.data.price = sc.actors[0].data.price
+        a.create_prediction()
+
     for t in list_ts:
         m.t = t
         for a in sc.actors:
+            a.get_market_schedule()
             order = a.generate_order()
             if order:
                 m.accept_order(order, callback=a.receive_market_results)
 
+        # Generate market maker order as ask
+        m.accept_order(
+            Order(1, t, 'market_maker', None, MARKET_MAKER_THRESHOLD,
+                  sc.actors[0].pred.price[0]))
+        # Generate market maker order as bid
+        m.accept_order(
+            Order(-1, t, 'market_maker', None, MARKET_MAKER_THRESHOLD,
+                  sc.actors[0].pred.selling_price[0]))
```

Comment:
> * hide behind market method m.add_market_maker(energy_price)
> 
>     * order sign (+/-): bid => `Order(+1,...` and ask => `Order(-1, ... `

actor.py states
```
Order = namedtuple("Order", ("type", "time", "actor_id", "cluster", "energy", "price"))
Order.__doc__ = """
Struct to hold order

:param type: sign of order, representing bid (-1) or ask (+1)
:param time: timestamp when order was created
:param actor_id: ID of ordering actor
:param energy: amount of energy the actor wants to trade. Will be rounded down(asks)/up(bids)
    according to the market's energy unit
:param price: bidding/asking price for 1 kWh
```
"""

Reply:
> further: central object, that holds market maker prices will be handled in future PR (issue #178)
> 
>     * proposal 1: sc.actors[0].pred.selling_price[0] + fee (from default or matrix)
> 
>     * proposal 2: distinct buying price

See above. Would like to change it once, when central access to selling and buying is finalized

Reply:
> > * hide behind market method m.add_market_maker(energy_price)
> >   
> >   * order sign (+/-): bid => `Order(+1,...` and ask => `Order(-1, ... `
> 
> actor.py states ...

My fault, I think was confused by the actor.pred.selling_price as it is a buying_price of the market_maker. We have to keep an eye on that when moving, this is also why it might make sense to rename it to mm_selling_price which would be used in the market_maker ask order